wordpress-mobile/WordPress-Android
# Don't keep errors for failed media that are being retried

Fixes #6796 by updating the FluxC hash
(actual fix is in https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/596)

To test:
1. start a draft
2. include some media
3. exit the editor, observe the progress notification going on
4. turn airplane mode ON
5. observe the error is shown in the Posts list.
6. turn airplane mode OFF
7. tap on Retry
8. observe the status label correctly changes to "Uploading..." along with the cloud icon, and the `Retry` button is gone as well.

cc @nbradbury
